lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -264,6 +265,12 @@ namespace Js
     BOOL JavascriptConversion::ToObject(Var aValue, ScriptContext* scriptContext, RecyclableObject** object)
     {
         Assert(object);
+        if (IsSimdType(aValue))
+        {
+            *object = scriptContext->GetLibrary()->CreateSIMDObject(aValue, JavascriptOperators::GetTypeId(aValue));
+            return TRUE;
+        }
```

Comment:
Can you please move this under default? I am not why this should be the first case as other primitive types are more common